rpc/client.go

```diff
@@ -45,10 +45,10 @@ type ClientPOS interface {
 // The response should be identical, regardless of the node queried,
 // as the information in question is objective and common to all nodes within a network.
 type ClientInformational interface {
-	// GetAccountBalance returns a purse's balance from a network.
+	// GetBalance returns a purse's balance from a network.
 	// The request takes in the formatted representation of a purse URef as a parameter.
 	// If the param stateRootHash is nil, the client will make an additional RPC call to retrieve the latest stateRootHash.
-	GetAccountBalance(ctx context.Context, stateRootHash *string, purseURef string) (StateGetBalanceResult, error)
+	GetBalance(ctx context.Context, purseURef string, stateRootHash *string) (StateGetBalanceResult, error)
```

Comment:
Just realized. Should we have `GetLatestBalance` and `GetBalanceByStateRootHash` methods?